arch/risc-v/include/elf.h

```diff
@@ -25,6 +25,16 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
+#define EM_ARCH  EM_RISCV
+
+#if defined(CONFIG_RISCV_TOOLCHAIN_GNU_RV64ILP32)
```

Comment:
don't use CONFIG_RISCV_TOOLCHAIN_GNU_RV64ILP32 in the common code I mention before